templates/package.json

```
@@ -57,8 +57,8 @@
     "react-redux": "5.0.7",
     "react-router-dom": "4.3.1",
     "react-router-redux": "5.0.0-alpha.9",
-    "react-scripts": "1.1.4",
-    "react-static": "^6.2.0",
+    "react-scripts": "2.1.3",
+    "react-static": "6.2.1",
     "react-static-plugin-react-router": "^6.1.1",
     "react-static-plugin-sass": "^6.1.1",
     "redux": "3.7.2",
```

I think u can update `redux`
```suggestion
    "redux": "4.0.1",
```

---

👍